docs/content/docs/connectors/table/formats/avro-glue.md

```diff
@@ -0,0 +1,191 @@
+
+Dependencies
+------------
+
+{{< sql_download_table "avro-glue" >}}
+
+How to create tables with Avro-Glue format
+--------------
+
+Example of a table using raw UTF-8 string as Kafka key and Avro records registered in the Schema Registry as Kafka values:
+
+```sql
+CREATE TABLE user_created (
+
+  -- one column mapped to the Kafka raw UTF-8 key
+  the_kafka_key STRING,
+  
+  -- a few columns mapped to the Avro fields of the Kafka value
+  id STRING,
+  name STRING, 
+  email STRING
+
+) WITH (
+
+  'connector' = 'kafka',
+  'topic' = 'user_events_example1',
+  'properties.bootstrap.servers' = 'localhost:9092',
+
+  -- UTF-8 string as Kafka keys, using the 'the_kafka_key' table column
+  'key.format' = 'raw',
+  'key.fields' = 'the_kafka_key',
+
+  'value.format' = 'avro-glue',
+  'value.avro-glue.region' = 'us-east-1',
+  'value.avro-glue.registry.name' = 'my-schema-registry',
+  'value.avro-glue.subject' = 'my-schema-name',
+  'value.fields-include' = 'EXCEPT_KEY'
+)
+```
+
+Format Options
+----------------
+
+Yes, these options have inconsistent naming convnetions.  No, I can't fix it.  This is for consistentcy with the existing [AWS Glue client code](https://github.com/awslabs/aws-glue-schema-registry/blob/master/common/src/main/java/com/amazonaws/services/schemaregistry/utils/AWSSchemaRegistryConstants.java#L20).
+
+<table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="text-left" style="width: 25%">Option</th>
+        <th class="text-center" style="width: 8%">Required</th>
+        <th class="text-center" style="width: 7%">Default</th>
+        <th class="text-center" style="width: 10%">Type</th>
+        <th class="text-center" style="width: 50%">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>format</h5></td>
+            <td>required</td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>Specify what format to use, here should be <code>'avro-glue'</code>.</td>
+        </tr>
+        <tr>
+            <td><h5>avro-glue.region</h5></td>
+            <td>required</td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>Specify what AWS region to use, such as <code>'us-east-1'</code>.</td>
+        </tr>
+        <tr>
+            <td><h5>avro-glue.registry.name</h5></td>
+            <td>required</td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>The name (not the ARN) of the Glue schema registry in which to store the schemas.</td>
+        </tr>
+        <tr>
+            <td><h5>avro-glue.subject</h5></td>
+            <td>required</td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>The subject name under which to store the schema in the registry.</td>
```

Comment:
Can this be named to `avro-glue.schema.name`?

Reply:
done